frame/transaction-payment/src/lib.rs

```diff
@@ -227,12 +237,18 @@ where
 }
 
 /// A struct to make the fee multiplier a constant
-pub struct ConstFeeMultiplier<M: Get<Multiplier>>(sp_std::marker::PhantomData<M>);
+pub struct ConstFeeMultiplier<M: Get<Multiplier>, X: Get<Multiplier>>(
+	sp_std::marker::PhantomData<M>,
+	sp_std::marker::PhantomData<X>,
+);
```

Comment:
```suggestion
pub struct ConstFeeMultiplier<M: Get<Multiplier>, X: Get<Multiplier>>(sp_std::marker::PhantomData<(M, X)>);
```

surprisingly common mistake